src/Database/PostgreSQL/PQTypes/SQL/Builder.hs

```diff
@@ -727,8 +728,16 @@ sqlWhereIsNULLE :: (MonadState v m, SqlWhere v, DBExtraException e, FromSQL a)
 sqlWhereIsNULLE exc col = sqlWhereEV (exc, col) $ col <+> "IS NULL"
 
 sqlWhereAny :: (MonadState v m, SqlWhere v) => [State SqlAll ()] -> m ()
-sqlWhereAny [] = sqlWhere "FALSE"
-sqlWhereAny l = sqlWhere $ "(" <+> smintercalate "OR" (map (parenthesize . toSQLCommand . flip execState (SqlAll [])) l) <+> ")"
+sqlWhereAny l = do
+  let sql = sqlForWhereAny l
+  sqlWhereE (DBBaseLineConditionIsFalse sql) sql
+
+sqlWhereAnyE :: (DBExtraException e, MonadState v m, SqlWhere v) => e -> [State SqlAll ()] -> m ()
```

Comment:
I'm aware that most top-level functions in this file lack Haddock comments, but would be nice if we starting adding them to the new ones at least.